packages/AutoTDD-GUI.package/ATDDBrowser.class/class/travisStatusFunctionalityDictionary.st

```diff
@@ -3,12 +3,12 @@ travisStatusFunctionalityDictionary
 	
 	^ OrderedDictionary
 		newFrom: {
-			'Add Travis Status' -> #addTravisStatus.
-			'Pause' -> #pauseSelectedTravisStatuses.
+			'Add Travis Status' -> #createTravisStatus.
+			'Pause' -> #pauseTravisStatuses.
 			'Resume' -> #resumeSelectedTravisStatuses.
 			'Run' -> #resumeSelectedTravisStatuses.
-      		'Edit' -> #editTravisStatus.
-			'Info' -> #openInfoForSelectedTravisStatuses.
-			'Results' -> #openResultsForSelectedTravisStatuses.
+      		'Edit' -> #editTravisStatuses.
+			'Info' -> #infoTravisStatuses.
+			'Results' -> #resultsTravisStatuses.
 			'Remove' -> #removeSelectedTravisStatuses.
```

Comment:
We should probably be consistent with periods on the last line of ordered dictionaries. For instance, this function does have a period on the last line, however the method `testingEntityResultsDictionary` from the same class does not. Perhaps there is something I'm not seeing?